x-pack/solutions/security/plugins/security_solution/common/endpoint/schema/actions.test.ts

```diff
@@ -1241,28 +1241,23 @@ describe('actions schemas', () => {
     });
 
     it('should throw if pid or entity id is used with type = kernel', () => {
-      // @ts-expect-error
```

Comment:
Thanks for updating the schema validation above. Consider updating the `.toThrow()` tests here to `.toThrowError('expected schema validation message')`.